.github/workflows/NightlyBaselineProfiles.yaml

```
@@ -0,0 +1,40 @@
+name: NightlyBaselineProfiles
+
+on:
+  schedule:
+    - cron:  '42 4 * * *'
+
+jobs:
+  baseline_profiles:
+    name: "Generate Baseline Profiles"
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
```

```suggestion
        uses: gradle/actions/wrapper-validation@v3
```
The latest version is `uses: gradle/actions/wrapper-validation@v3` check this [documentation](https://github.com/gradle/actions/blob/main/docs/wrapper-validation.md#usage)
Is there any reason to use old version validation action?